components/Navbar.tsx

```diff
@@ -24,7 +26,13 @@ import {
   NavbarLinkWhyUse
 } from "./NavbarComponents"
 
-export const MobileNav: React.FC<React.PropsWithChildren<unknown>> = () => {
+export const MainNavbar: React.FC<React.PropsWithChildren<unknown>> = () => {
```

Comment:
brought both MobileNav and DesktopNav into one file so that future coders know to update both in the future